app/views/clients/show.html.erb

```diff
@@ -11,8 +11,8 @@
     <h3>Phone for texting: <%= @client.sms_phone_number %></h3>
   </div>
   <h2>Contact History</h2>
-  <% if @contact_history.present? %>
-    <ul class="contact-history">
+    <ul id="contact-history" aria-live="polite">
```

Comment:
DOM element `#contact-history` get replaced with new data as it streams in. For accessibility, [aria-live=\"polite\"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) defines a Live Region and notifies user of updates.